Foundation/NSLocale.swift

```diff
@@ -109,39 +109,39 @@ extension NSLocale {
     open class var isoLanguageCodes: [String] {
         var identifiers = Array<String>()
         for obj in CFLocaleCopyISOLanguageCodes()._nsObject {
-            identifiers.append((obj as! NSString)._swiftObject)
+            identifiers.append(obj as! String)
```

Comment:
Isn't the result type here `CFStringRef`, which is not directly bridgeable to `Swift.String`?

Reply:
Nope, they are `Swift.String`s. If you try to get any of that properties in the actual Swift Foundation implementation they will throw with the following execution:

    Could not cast value of type 'Swift.String' (0x7ffff3e61970) to 'Foundation.NSString' (0x7ffff75a9190).

Reply:
To prevent this kin of confusion, can we just replace the whole thing with:

```swift
_SwiftValue.fetch(CFLocaleCopyISOLanguageCodes()) as? [String] ?? []
```

That's the least confusing primitive for bridging on Linux.

Reply:
Great! I'll commit the changes.